src/Controller/AccountController.java

```diff
@@ -130,6 +130,7 @@ public void validateNumber()
         if (account_no.getText().trim().isEmpty())
             this.submit.setDisable(true);
         else
+            this.account_no.setStyle("-fx-text-box-border:red;");
```

Comment:
This is a bug.  You have added a line to the else which now results in the next line (`this.handleChange();`) being executed unconditionally.  You need wrap the two statements in curly braces.